ci/get-ocp-repo.sh

```diff
@@ -1,205 +1,19 @@
-    manifest="$cosa_workdir/src/config/manifest.yaml"
-    if [ -f "$cosa_workdir/src/config.json" ]; then
-        variant="$(jq --raw-output '."coreos-assembler.config-variant"' 'src/config.json')"
-        manifest="$cosa_workdir/src/config/manifest-${variant}.yaml"
-    fi
-
-    # flatten manifest and query a couple of fields
-    json=$(rpm-ostree compose tree --print-only "$manifest")
-    osname=$(jq -r '.metadata.name' <<< "$json")
-    is_ocp_variant=$(jq '.packages | contains(["cri-o"])' <<< "$json")
-
-    if [ "$osname" = scos ] && [ "$is_ocp_variant" = false ]; then
-        # this is the pure SCOS case; we don't need any additional repos at all
-        info "Building pure SCOS variant. Exiting..."
-        exit 0
-    elif [ "$osname" = scos ]; then
-        # We still need the OCP repos for now unfortunately because not
-        # everything is in the Stream repo. For the RHEL version, just use the
-        # default variant's one.
-        json=$(rpm-ostree compose tree --print-only "$cosa_workdir/src/config/manifest.yaml")
-    fi
-    version=$(jq -r '.["automatic-version-prefix"]' <<< "$json")
-    if [ "$is_ocp_variant" = true ]; then
-        # RHEL version is second field
-        info "Building OCP variant"
-        rhel_version=$(cut -f2 -d. <<< "$version")
-    else
-        # RHEL version is first and second field
-        info "Building pure variant"
-        rhel_version=$(cut -f1-2 -d. <<< "$version")
-        rhel_version=${rhel_version//./}
-    fi
-    info "Got RHEL version $rhel_version from automatic-version-prefix value $version"
-
-    if [ -z "$output_dir" ]; then
-        output_dir="$cosa_workdir/src/config"
-    fi
-fi
-
-mkdir -p "$output_dir"
-repo_path="$output_dir/ocp.repo"
-
-set -x
-curl --fail -L "http://base-${ocp_version}-rhel${rhel_version}.ocp.svc.cluster.local" -o "$repo_path"
-set +x
-
-if [ "${rhel_version}" = 96 ]; then
-    # XXX: also currently also add 9.4 repos for crun-wasm when building extensions
+urls=(
+    # theoretically that's the only one we need
+    "http://base-4-19-rhel96.ocp.svc.cluster.local"
+    # XXX: but also currently add 9.4 repos for crun-wasm when building extensions
     # https://github.com/openshift/os/issues/1680
     # https://github.com/openshift/os/pull/1682
     # https://issues.redhat.com/browse/COS-3075
-    curl --fail -L http://base-4-19-rhel94.ocp.svc.cluster.local >> "$repo_path"
-fi
+    "http://base-4-19-rhel94.ocp.svc.cluster.local"
+)
 
-# If we're building the SCOS OKD variant, then strip away all the RHEL repos and just keep the plashet.
-# Temporary workaround until we have all packages for SCOS in CentOS Stream.
-if [ "$osname" = scos ]; then
-    info "Neutering RHEL repos for SCOS"
-    awk '/server-ose/,/^$/' "$repo_path" > "$repo_path.tmp"
-    # only pull in certain Openshift packages as the rest come from the c9s repo
-    sed -i '/^baseurl = /a includepkgs=openshift-* ose-aws-ecr-* ose-azure-acr-* ose-gcp-gcr-*' "$repo_path.tmp"
```

Comment:
without this `includepkgs` line, rpms like cri-o, conmon, cri-tools will be pulled in from the rhel repo as well which we do not want  for OKD.

Reply:
Fixed!